DFE-Digital/academies-academisation-api
# 105888 add additional info data storage

Adding additional information data capture onto GET & PUT endpoints as per this ticket:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/105888?McasTsid=26110

## Type of change

## Expected behaviour
All additional information properties to be present in GET && PUT endpoint
All additional information properties to be present in Legacy GET endpoint
All additional information properties table in sip DB

## Steps to reproduce issue (if relevant)
n/a

## Steps to test this PR
1. Run unit tests to ensure nothing broken
2. Run dotnet ef database update - to ensure new column created
3. run API to ensure new properties are available on GET && PUT endpoints
4. run API to ensure new properties are available on legacy GET endpoint

## Prerequisites
n/a
